src/verinfast/config.py

```diff
@@ -213,6 +217,10 @@ def __init__(self, cfg_path: str = None) -> None:
                     parent = curr_path.parent
                     curr_path = parent
         self.handle_config_file()
+        if self.config is not FileNotFoundError:
```

Comment:
this is redundant to 226

Reply:
It's not. On line 225, handle_args gets called which may modify self.config.

So orig_config stores the config from the file, but self.config ends up being a combination of what was in the file plus arg changes. I wanted to see both in the logs.

Reply:
You were correct.